docs/source/onnxruntime/usage_guides/trainer.mdx

```diff
@@ -10,14 +10,43 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 specific language governing permissions and limitations under the License.
 -->
 
-# Trainer
+# How to accelerate training with ONNX Runtime
 
-The [`ORTTrainer`] and [`ORTSeq2SeqTrainer`] classes provide APIs for training PyTorch models with __[ONNX Runtime (ORT)](https://onnxruntime.ai/)__.
-Taking ONNX Runtime as backend, [`ORTTrainer`] and [`ORTSeq2SeqTrainer`] optimize the computation graph and the memory usage. They also support
-mixed precision training implemented by ORT, as well as distributed training on multiple GPUs. With them, you will be able to achieve
-__lower latency, higher throughput, and larger maximum batch size__ while training large transformers models.
+Optimum integrates ONNX Runtime Training through an `ORTTrainer` API that extends `Trainer` in [Transformers](https://huggingface.co/docs/transformers/index).
+With the solution, training time can be improved by greater than 35% for many popular Hugging Face models compared to PyTorch under eager mode.
 
-## Prerequisite
+[`ORTTrainer`] and [`ORTSeq2SeqTrainer`] APIs make it easy to compose __[ONNX Runtime (ORT)](https://onnxruntime.ai/)__ with other features in [`Trainer`](https://huggingface.co/docs/transformers/main_classes/trainer).
+It contains feature-complete training loop and evaluation loop, and supports hyperparameter search, mixed-precision training and distributed training with multiple [NVIDIA](https://techcommunity.microsoft.com/t5/ai-machine-learning-blog/accelerate-pytorch-transformer-model-training-with-onnx-runtime/ba-p/2540471)
+and [AMD](https://cloudblogs.microsoft.com/opensource/2021/07/13/onnx-runtime-release-1-8-1-previews-support-for-accelerated-training-on-amd-gpus-with-the-amd-rocm-open-software-platform/) GPUs.
+With ONNX Runtime backend, [`ORTTrainer`] and [`ORTSeq2SeqTrainer`] take advantage of:
```

Comment:
```suggestion
Through the ONNX Runtime Training backend, [`ORTTrainer`] and [`ORTSeq2SeqTrainer`] take advantage of:
```

Reply:
Do you want to be precise about the ort backend here is specified for training?

Reply:
`With ONNX Runtime backend` just sound not super good. Maybe with "the"? I don't know

Reply:
Added.